src/components/twilio-voice-monitoring/public/twilio-voice-monitoring.js

```diff
@@ -106,37 +118,96 @@ class TwilioVoiceMonitoring extends HTMLElement {
     this.#callSid = undefined;
   };
 
+  #setDeviceHandlers(device) {
+    device.on('destroyed', () => {
+      this.#log('INFO[Device]', 'destroyed');
+    });
+    device.on('error', (twilioError, call) => {
+      const errorLog = {
+        callSid: call.parameters.CallSid,
+        code: twilioError.code,
+        message: twilioError.message,
+      };
+      this.#log('Error[Device]', JSON.stringify(errorLog, null, 2));
+    });
+    device.on('incoming', (call) => {
+      this.#log('INFO[Device]', `incoming call from: ${call.parameters.From}`);
+    });
+    device.on('registered', () => {
+      this.#log('INFO[Device]', 'registered');
+    });
+    device.on('registering', () => {
+      this.#log('INFO[Device]', 'registering');
+    });
+    device.on('tokenWillExpire', () => {
+      this.#log('INFO[Device]', 'refreshing token');
```

Comment:
```suggestion
      this.#log('INFO[Device]', 'tokenWillExpire');
```
The implication might be that we want to refresh the token, but that's not _exactly_ happening here yet. We can add a log to the token refresh logic if we want to make that verbose.